src/modules/CreateChannel/context/CreateChannelProvider.tsx

```diff
@@ -39,7 +39,12 @@ export interface CreateChannelContextInterface {
   createChannel: CreateChannel;
   sdk: SendbirdGroupChat;
   userListQuery?(): UserListQuery;
+  /**
+   * @deprecated
+   * Use the onClickCreateChannel instead
+   */
```

Comment:
If you haven't actually deprecated it, it would be a good idea to add comments later when you actually work on it.